code/modules/admin/player_panel.dm

```diff
@@ -348,12 +348,22 @@
 		var/living_players_antagonist = 0
 		var/brains = 0
 		var/other_players = 0
+		var/living_skipped = 0
+		var/ctf_mobs = 0
 		for(var/mob/M in mob_list)
+			if(ishuman(M))
+				var/mob/living/carbon/human/H = M
+				if(istype(H.wear_suit, /obj/item/clothing/suit/space/hardsuit/shielded/ctf))

```

Comment:
This is a really ugly check. If you are already skipping mobs on z2 you don't need to check for ctf mobs.